src/client/components/pages/entities/creator.js

```diff
@@ -53,23 +63,27 @@ function CreatorAttributes({creator}) {
 					<dl>
 						<dt>Type</dt>
 						<dd>{type}</dd>
-						<dt>Gender</dt>
-						<dd>{gender}</dd>
+						{showGender ?
+							<React.Fragment>
+								<dt>Gender</dt>
+								<dd>{gender}</dd>
+							</React.Fragment> : null
+						}
 					</dl>
 				</Col>
 				<Col md={3}>
 					<dl>
-						<dt>Begin Date</dt>
+						<dt>{beginDateLabel}</dt>
 						<dd>{beginDate}</dd>
-						<dt>Begin Area</dt>
+						<dt>{beginAreaLabel}</dt>
 						<dd>{beginArea}</dd>
 					</dl>
 				</Col>
 				<Col md={3}>
```

Comment:
I merged in another PR recently, but I don't like that this section appears if the author is alive.

Could you please add again the `{ creator.ended && ...}` that was there before that hid the column?
See [the change in question](https://github.com/bookbrainz/bookbrainz-site/commit/7c0e82c2d9fdb7112db7331bb2fa16be2c825d26#diff-c08fec8c7f9f07575b69c6361f9e0bc8L66)